### ISSUE TITLE
Console Error

### ISSUE THREAD

**Issue by tomasmessing** [konsolas/AAC-Issues#650]
### 001
```
[14:27:00 INFO]: [Bank] Saving Bank for 36cd0809-dc80-38b4-ae54-ff61161f0e46 to file
[14:27:00 INFO]: [Bank] Saved Bank for 36cd0809-dc80-38b4-ae54-ff61161f0e46 to file
[14:27:00 INFO]: casasta issued server command: /home voxel
[14:27:00 INFO]: LUKYS123 issued server command: /warp rybar
[14:27:00 INFO]: Freddy_LP_CZ issued server command: /home
[14:27:01 INFO]: LUKYS123 issued server command: /warp rybar
[14:27:01 INFO]: casasta issued server command: /ah
[14:27:01 INFO]: LUKYS123 issued server command: /warp rybar
[14:27:02 INFO]: LUKYS123 issued server command: /warp rybar
[14:27:02 INFO]: LUKYS123 issued server command: /warp rybar
[14:27:03 INFO]: LUKYS123 issued server command: /warp rybar
[14:27:03 INFO]: Kukyn issued server command: /ec
[14:27:03 INFO]: Hraci Kukyn byl zablokovan prikaz.
[14:27:03 INFO]: LUKYS123 issued server command: /warp rybar
[14:27:04 INFO]: Jarl_Azuls issued server command: /ah
[14:27:04 INFO]: LUKYS123 issued server command: /warp rybar
[14:27:04 INFO]: LUKYS123 issued server command: /warp rybar
[14:27:05 INFO]: LUKYS123 issued server command: /warp rybar
[14:27:06 INFO]: Baron Vanilkaman » a jaky mas to oenchantovane nebo normalni???
[14:27:06 INFO]: Kukyn issued server command: /tpa TermixGame
[14:27:08 ERROR]: Could not pass event PlayerMoveEvent to AAC v3.1.5
org.bukkit.event.EventException
at org.bukkit.plugin.java.JavaPluginLoader$1.execute(JavaPluginLoader.java:306) ~[minecraft_server.jar:git-Spigot-54ec0b8-473febc]
at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:62) ~[minecraft_server.jar:git-Spigot-54ec0b8-473febc]
at org.bukkit.plugin.SimplePluginManager.fireEvent(SimplePluginManager.java:502) [minecraft_server.jar:git-Spigot-54ec0b8-473febc]
at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:487) [minecraft_server.jar:git-Spigot-54ec0b8-473febc]
at net.minecraft.server.v1_11_R1.PlayerConnection.a(PlayerConnection.java:615) [minecraft_server.jar:git-Spigot-54ec0b8-473febc]
at net.minecraft.server.v1_11_R1.PacketPlayInFlying.a(SourceFile:126) [minecraft_server.jar:git-Spigot-54ec0b8-473febc]
at net.minecraft.server.v1_11_R1.PacketPlayInFlying$PacketPlayInLook.a(SourceFile:88) [minecraft_server.jar:git-Spigot-54ec0b8-473febc]
at net.minecraft.server.v1_11_R1.PlayerConnectionUtils$1.run(SourceFile:13) [minecraft_server.jar:git-Spigot-54ec0b8-473febc]
at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [?:1.8.0_74]
at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:1.8.0_74]
at net.minecraft.server.v1_11_R1.SystemUtils.a(SourceFile:46) [minecraft_server.jar:git-Spigot-54ec0b8-473febc]
at net.minecraft.server.v1_11_R1.MinecraftServer.D(MinecraftServer.java:747) [minecraft_server.jar:git-Spigot-54ec0b8-473febc]
at net.minecraft.server.v1_11_R1.DedicatedServer.D(DedicatedServer.java:399) [minecraft_server.jar:git-Spigot-54ec0b8-473febc]
at net.minecraft.server.v1_11_R1.MinecraftServer.C(MinecraftServer.java:678) [minecraft_server.jar:git-Spigot-54ec0b8-473febc]
at net.minecraft.server.v1_11_R1.MinecraftServer.run(MinecraftServer.java:576) [minecraft_server.jar:git-Spigot-54ec0b8-473febc]
at java.lang.Thread.run(Thread.java:745) [?:1.8.0_74]
Caused by: java.lang.IllegalArgumentException: Cannot measure distance between world and novyspawn
at org.bukkit.Location.distanceSquared(Location.java:456) ~[minecraft_server.jar:git-Spigot-54ec0b8-473febc]
at org.bukkit.Location.distance(Location.java:439) ~[minecraft_server.jar:git-Spigot-54ec0b8-473febc]
at me.konsolas.aac.aS.a(aS.java:21) ~[?:?]
at me.konsolas.aac.aS.a(aS.java:7) ~[?:?]
at me.konsolas.aac.bp.a(bp.java:142) ~[?:?]
at sun.reflect.GeneratedMethodAccessor160.invoke(Unknown Source) ~[?:?]
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_74]
at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_74]
at org.bukkit.plugin.java.JavaPluginLoader$1.execute(JavaPluginLoader.java:302) ~[minecraft_server.jar:git-Spigot-54ec0b8-473febc]
... 15 more
```
### 002
`[14:45:49 WARN]: [AAC] Task #1176278 for AAC v3.1.6 generated an exception
java.lang.IllegalArgumentException: Cannot measure distance between world and novyspawn
at org.bukkit.Location.distanceSquared(Location.java:456) ~[minecraft_server.jar:git-Spigot-54ec0b8-473febc]
at org.bukkit.Location.distance(Location.java:439) ~[minecraft_server.jar:git-Spigot-54ec0b8-473febc]
at me.konsolas.aac.z.a(z.java:95) ~[?:?]
at me.konsolas.aac.z.a(z.java:68) ~[?:?]
at me.konsolas.aac.cG.a(cG.java:38) ~[?:?]
at me.konsolas.aac.bO.lambda$new$4(bO.java:8) ~[?:?]
at org.bukkit.craftbukkit.v1_11_R1.scheduler.CraftTask.run(CraftTask.java:71) ~[minecraft_server.jar:git-Spigot-54ec0b8-473febc]
at org.bukkit.craftbukkit.v1_11_R1.scheduler.CraftScheduler.mainThreadHeartbeat(CraftScheduler.java:353) [minecraft_server.jar:git-Spigot-54ec0b8-473febc]
at net.minecraft.server.v1_11_R1.MinecraftServer.D(MinecraftServer.java:738) [minecraft_server.jar:git-Spigot-54ec0b8-473febc]
at net.minecraft.server.v1_11_R1.DedicatedServer.D(DedicatedServer.java:399) [minecraft_server.jar:git-Spigot-54ec0b8-473febc]
at net.minecraft.server.v1_11_R1.MinecraftServer.C(MinecraftServer.java:678) [minecraft_server.jar:git-Spigot-54ec0b8-473febc]
at net.minecraft.server.v1_11_R1.MinecraftServer.run(MinecraftServer.java:576) [minecraft_server.jar:git-Spigot-54ec0b8-473febc]
at java.lang.Thread.run(Thread.java:745) [?:1.8.0_74]`

**Comment by Janmm14** [comment]
Usually those errors just indicate a missing check which would abort computation anyway. Might be already fixed in AAC 3.1.6, but its recommended to stay on 3.1.5

**Comment by konsolas** [comment]
-b3
